plugins/pom.xml

```diff
@@ -75,10 +75,9 @@
         <module>pur</module>
         <module>ms-access</module>
         <module>postgresql-db-bulk-loader</module>
+        <module>oracle-bulk-loader</module>
         <module>xml</module>
         <module>streaming</module>
-        <module>metastore-locator</module>
```

Comment:
I think you need to rebase one last time; we merged in more changes and these two modules should not get removed from this assembly here.

Reply:
done